conmon-rs/server/src/attach.rs

```diff
@@ -237,7 +237,11 @@ impl Attach {
                                 e.raw_os_error().context("get OS error")?
                             ),
                         },
-                        _ => {}
+                        _ => {
+                            debug!("Stopping read loop because there is nothing more to read");
```

Comment:
I am not sure about this. I think we should double check with a manual check with crictl to make sure it behaves as expected

Reply:
Hm, I tested it with CRI-O and it seems to work with crictl. Can we verify if the change also works with Podman somehow?

Reply:
looks like we are missing a match statement when `n==0`

Reply:
Yep, that's what this path is for :+1: